spec/presenters/video_presenter_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+describe VideoPresenter do
+  
+  describe :display_large_video_image do
+    
+    it "should display a large video image if one is available" do
```

Comment:
Word `should` in message is so common that new style rather prefers using 3rd person already `it displays`. I suggest to get rid of the habit of `should`.